tests/slow_retrieval_server.py

```diff
@@ -62,38 +54,18 @@ def do_GET(self):
       self.send_header('Content-length', str(len(data)))
       self.end_headers()
 
-      if self.server.test_mode == 'mode_1':
-        # Before sending any data, the server does nothing for a long time.
-        DELAY = 40
-        time.sleep(DELAY)
-        self.wfile.write(data)
-
-        return
-
-      # 'mode_2'
-      else:
-        DELAY = 1
-        # Throttle the file by sending a character every DELAY seconds.
```

Comment:
It would be useful to update the commit message to make it clear that only the throttle/mode_2 slow-retrieval test has been removed and that the delayed/mode_1 slow retrieval test remains